packages/ibm-products/src/components/BigNumbers/BigNumbers.mdx

```diff
@@ -33,12 +33,6 @@ of a button as well as tool tip functionality. The default locale is English
   <Story of={BigNumbersStories.bigNumbers} />
 </Canvas>
 
-### Big numbers with edit button
```

Comment:
Removed story; this feature is now available through the default story **Controls**.